docs/source/user_guide/users.ipynb

```diff
@@ -0,0 +1,1602 @@
lEgo2hkY+wdI+4/6W6GcJtUzuMUUEABBRRQQIF1BHL837fKeMDbr7NvPn3t0gB89Qj7TrJLpv/lkke91y8nfhjCbRJRj1FAAQUUUEABBQYIXK+M68tLq1sOeL7+0C1L4+xf6w/O4HZeWj4QaIZwOwK45wzSNwkFFFBAAQUUUECBhkA2vrLH7RmNx5t371b2+5fmE1PcN4TbFHgeqoACCiiggAIKTCqQEy6OATKqxpWHJPLA0gB82JB9Rn3qSsAnS3r1y72fM4TbqITup4ACCiiggAIKTCeQ8YGzITZshu/jyj5/O0VWa4VwOwq41xTpeqgCCiiggAIKKKDABAL/Vxp4a427e3Z5fpQJI4OyvxPwi5JG1ev3lzL+75KDDvAxBRRQQAEFFFBAgfkK7AGcDWS4t20GZPXy0ni74YDnhj00LITbuGkNy8fnFFBAAQUUUEABBSYQeElp5L1owLH/UZ67yoDnBj1UhXA7udHrlwtQ51p/GVnETQEFFFBAAQUUUGDJAtnz9xvgHOAajbK8rzTkdmo8PujuTYFDGg2/jD+c0T12HnSAjymggAIKKKCAAgosT+CupeH2+UYRPl0Wjc6evbW2HMs3KITbT4DbrHWQjyuggAIKKKCAAgosX+BjpRH4gFpRvgn8uXa/fjMv5+Zl3eMavX65/37AFvWdva2AAgoooIACCijQPoHdgdNKg66aoZvRQn4/oKgZwu0rjYZfzvI1hNsALB9SQAEFFFBAAQXaLFAt+/KaUsijgbyUW22XGBLC7R7VTv5XQAEFFFBAAQUU6I5ALtp8GJBr9d2o9Ah+vRT/fkAu3lyt55f/cwmZ/ddYQqY7tbakCiiggAIKKKDAigvkxI2cvfu98v+La4RwOwi4+opbWX0FFFBAAQUUUKA3Au+t9fSdV7udvX45JtAQbr051VZEAQUUUEABBRTYIJAzgXNdwPrlXkO4+epQQAEFFFBAAQV6KLAr8MFGwy8bgd8CDOHWwxNulRRQQAEFFFBgdQUM4ba6596aK6CAAgoooMAKCtwMOLTR63d+We7FEG4r+IKwygoooIACCijQX4FdysLN2dirj/X7MXDr/lbbmimggAIKKKCAAqsnYAi31Tvn1lgBBRRQQAEFVljgOsBXGz1+2fuXIdwut8IuVl0BBRRQQAEFFOidwFoh3A4H9uldba2QAgoooEDvBfJylpsCCqwtsC/wWuDytV0yhNsryt9ZtcfHvbkncC1gdyAbmfl+PK2EjMvG5S/GTdD9FVBAAQUUUEABBSYXuPIaIdw+O2UIt72BtwJ/GHApuT6ZJG+fALwNuOnk1fBIBRRQQAEFFFBAgfUELg7sD5zZaKBNG8LtasBnGmk2G3zD7uexmYabAgoooIACCiigwAwF7gz8stFIy1i+BwDbTZHPQwc0KIc19tZ6LhulD56iHB6qgAIKKKCAAgooUATWCuH2TeAGUyo9t9GgrDfucozfy4GHAXcB7lRuvxTIMYD1feu3nzVlmTxcAQUUUEABBRRYWYEqhNspjcZWjr3LXrtpJ0n9cyPdqhF3GHDbEdRvBfxkjTQePcLx7qKAAgoooIACCihQE7j5kBBuO9X2m/Rmpn/ugMbbO4Ctxkh0S+DtA9I5B7jxGOm4qwIKKKCAAgoosLICiwjhlo22nw5otOWC0ZP0KuYxeWzVg1j9/wGwxcqeSSuugAIKKKCAAgqsI5CNqMcAf2o0pE4F9ptxQ+pxjTyywfazMXv+mtXJXsOfD0j3n5o7el8BBRRQQAEFFFAAFhnCLRuaRzQaaheMOOZvvXN1u0a62bDMRqGbAgoooIACCiigQBHYHjhwwFi8nGF7hzkpZWi46hJt9f9LM8xrUCzi28wwfZNSQAEFFFBAAQU6K5CzeI9uNMYybFsuoTLOJIxxAd7YyDMbgbNcu+/hA9J/9biFdH8FFFBAAQUUUKBPAldZI4TboiJpNMfp5UzgbWcInDGD/9JoBP5ohumblAIKKKCAAgoo0BmBtUK4HQnca0G1yMZZjverLv3m/5ypO+utuTZgRivZetaZmJ4CCiiggAIKKNBmgYyi0QzhluvkZUzfaUK4jVvnvRqNv2wA5hp+s97ePSCf6886E9NTQAEFFOivwMX6WzVrtgICVyiXez83ZxFlAAAN+klEQVQH7PH/27uzGFmqMg7gf1RUFmURRRRQQUUQFGR3RUFxARMXYmLQxMRgiFF8Q18ML0ZeNLkPGpdoDMYXfLzBFVzBhUUUlVUlqAiC7KAooObT7tCcW32n73TP3KqeXyWT6a6pOnXOr+5NvtRy/hPjrQi3o0cF4AMT69f6434dB7ixY928q+ot43YpCwsBAgQIEJhJQAE4E5ONeiYwjnCryZZPmejb7aMs3Xor9qqJ9ev1cc+OA1XM3KKXmruwXfZqV/hOgAABAgSmCUgRmCZjfV8Fjk/y2SSHT3Swnrurt2/rlu9dE+vX++NOHQfsKtY6NtumVV1tdh17mxq1MQECBAhsHAEF4MY510MfaV3h+vRoSpXJOLXLkpyZ5IqeDrCeA1z00tVmFcEWAgQIECAwk4BbwDMx2Wg7Cowj3K5L8p6JLN17k3wkyct7VPzVPIPtslu7YgHfu9q8bwHtaoIAAQIENoiAK4Ab5EQPdJiHjm73tkkXXx1N6HxLz8Z1Z0d/du9YN++qrjarILYQIECAAAECBAYrMI5wq/nt6nbn+OfqNYxwWwRWvXk87uv49+cW0XDTxpc6jnNIs42vBAgQIECAAIHBCFSE281NgTOOcNux56OowrWdCPryNejzLxufShvpu80aMGiSAAECBAgQGLrAQUkubAqbuop2QZIDBjS4elZxfPWvfteE1ItM6ai3fdsro6LgBvQPRFcJECBAgACBpA8Rbos8D59pCsAqAt+1wAO8u6P9Ty6wfU0RIECAAAECBNZU4OQkNzQFzTjCbec1PfLaNX5SM54qACupZFFL11XSmhvRQoAAAQIECBDotUBFpm3uKJQuT
```

Comment:
cuSpatial (capital S).  As I commented before, this sentence seems redundant since you just discussed that other join algorithm above. At least link to that section.  Also, here's another `point_in_polygon_quadtree` that should be `quadtree_point_in_polygon`.d